packages/nuxt/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "nuxt",
+  "version": "1.0.0",
+  "description": "nuxt",
+  "author": "Przemysław Spaczek",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt build && nuxt start",
+    "generate": "nuxt generate",
+    "test": "jest",
+    "storybook": "nuxt-storybook",
+    "build-storybook": "nuxt-storybook build"
+  },
+  "dependencies": {
+    "@nuxt/babel-preset-app": "^2.14.3",
+    "@nuxtjs/pwa": "^3.0.0-0",
+    "@storefront-ui/vue": "0.10.6",
+    "lerna": "^3.16.4",
+    "nuxt": "1.0.0",
```

Comment:
This uses very old version of nuxt. I think it would be reasonable to have a newer version (i.e. 2.15) but it can be done later if it will produce many errors.

Reply:
Maybe I should upgrade it now, because there is some issue conneted with nuxt in gh actions workflow. I still cannot run it properly when pushing to remote repo.